dogpile/cache/backends/memcached.py

```diff
@@ -488,6 +533,22 @@ class PyMemcacheBackend(GenericMemcachedBackend):
     :param serde: optional "serde". Defaults to
      ``pymemcache.serde.pickle_serde``
     :param default_noreply: Defaults to False
+    :param socket_keepalive: optional socket keepalive, will be used for
```

Comment:
**mike bayer** (zzzeek) wrote:

these params all need "..versionadded:: 1.1.5"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2965